spyder/widgets/variableexplorer/dataframeeditor.py

```diff
@@ -12,16 +12,18 @@
 """
 Pandas DataFrame Editor Dialog
 """
-
+import time

```

Comment:
This needs to be

```python
"""
Pandas DataFrame Editor Dialog
"""

# Std imports
import time

# Third party imports
...
```

Reply:
Also, please add in the comment above this line a copyright to the `gtabview` project, with its corresponding license.